exchanges.yml

```diff
@@ -358,6 +358,16 @@ components:
           description: An optional step directive that tells the exchange to handle challenge management via a VC API verifier service it has a zcap for.
         verifiablePresentationRequest:
           $ref: "./components/VerifiablePresentationRequest.yml#/components/schemas/VerifiablePresentationRequest"
+        presentationSchema:
+          type: object
+          description: The presentation schema to use when validating a presentation in the current step.
+          properties:
+            types:
+              type: string
+              description: The type of presentation schema, such as <code>JsonSchema</code>.
+            jsonSchema:
+              type: object
+              description: The JSON Schema object to use if the <code>type</code> is <code>JsonSchema</code>.
```

Comment:
While we're here, we might want to also add the other optional counterpart to this schema, which is a schema to run against the result of verifying the presentation, `verifyPresentationResponseSchema`. This schema is also optional and can be used on the *result* from a workflow step that uses a VC API verifier to "verify a presentation". It has the same structure and is just a different schema that executes at a different time on a different value (on the JSON response from https://w3c-ccg.github.io/vc-api/#verify-presentation).

Reply:
Fixed in 933a6ee42848132c10b5200cf6d86ee1a79a77a8.